### PR TITLE
Fixed Companies auto complete

### PR DESCRIPTION
**Pull request by raymondlam12**
Self explanatory

### CODE REVIEW

File: client/Review/ReviewForm.jsx [hunk 1]
```diff
@@ -41,7 +41,20 @@ function ratingRange(pos, rating) {
...
+  let companies = [];
+
+  fetch('api/company')
+    .then((response) => {
+      return response.json();
+    })
+    .then((data) => {
+      console.log(data);
+      data.forEach(d => {
+        companies.push(d["name"]);
```
Comment (ggjack) [inline client/Review/ReviewForm.jsx]: nit: was seeing duplicates, one had whitespace at the end. Probably gonna add `trim()`

**Comment by ggjack** [comment]
Tested LGTM

File: client/Review/ReviewForm.jsx (hunk 1 above)
Comment (ggjack) [reply client/Review/ReviewForm.jsx]: oh, we assume its already unique in db and just get a list of the names.
Also the INSERT already trims ¯\_(ツ)_/¯

Comment (raymondlam12) [reply client/Review/ReviewForm.jsx]: yeah im just going to assume that database is sound

**Pull request merged**